rcsbsearchapi/search.py

```diff
@@ -406,15 +411,15 @@ class Attr:
     attribute: str
     type: Optional[str] = STRUCTURE_ATTRIBUTE_SEARCH_SERVICE  # this will be changed later, this is to allow the program to still run. Will not be optional.
 
-    def exact_match(self, value: Union[str, "Value[str]"]) -> Terminal:
+    def exact_match(self, value: Union[str, "Value[str]"]) -> AttributeQuery:
         """Exact match with the value"""
         if isinstance(value, Value):
             value = value.value
-        return Terminal(self.attribute, self.type, "exact_match", value)
+        return AttributeQuery(self.attribute, "exact_match", value, self.type)
```

Comment:
Hmm, is it possible to move `self.type` back to be the second argument? I wonder if you had started from an old version of the repo by accident...

Reply:
On second thought, you can probably leave this as is, since `type` defaults to structure attribute search (right?). So, don't worry about reverting this for now.

Reply:
`type` does default to structure attribute search.